packages/app/src/navigation/SidebarNavigation.vue

```diff
@@ -178,18 +177,13 @@ watchEffect(() => {
     return
   }
 
-  const showNewBadge = isAllowedFeature('debugNewBadge', loginConnectStore)
-  const newBadge: Badge = { value: t('sidebar.debug.new'), status: 'success', label: t('sidebar.debug.debugFeature') }
-
   if (props.gql?.currentProject?.cloudProject?.__typename === 'CloudProject'
     && props.gql.currentProject.cloudProject.runByNumber
     && props.online
   ) {
     const { status, totalFailed } = props.gql.currentProject.cloudProject.runByNumber || {}
 
     if (status === 'NOTESTS' || status === 'RUNNING') {
```

Comment:
I think so, because otherwise we'll hit this code block for the `NOTESTS` and `RUNNING` statuses
```
setDebugBadge({ value: countToDisplay, status: 'error', label: errorLabel })
```
Which I don't think we want